cmd/draft/plugin.go

```diff
@@ -125,24 +161,37 @@ func manuallyProcessArgs(args []string) ([]string, []string) {
 	return known, unknown
 }
 
-// findPlugins returns a list of YAML files that describe plugins.
-func findPlugins(plugdirs string) ([]*plugin.Plugin, error) {
-	found := []*plugin.Plugin{}
-	// Let's get all UNIXy and allow path separators
-	for _, p := range filepath.SplitList(plugdirs) {
-		matches, err := plugin.LoadAll(p)
-		if err != nil {
-			return matches, err
+// runHook will execute a plugin hook.
+func runHook(p *plugin.Plugin, event string) error {
+	hook := p.Metadata.Hooks.Get(event)
+	if hook == "" {
+		return nil
+	}
+
+	prog := exec.Command("sh", "-c", hook)
+	// TODO make this work on windows
+	// I think its ... ¯\_(ツ)_/¯
+	// prog := exec.Command("cmd", "/C", p.Metadata.Hooks.Install())
```

Comment:
I can see if I can round up a Windows VM to test future PRs but I wouldn't fret too much. We can iron out the Windows kinks once we get someone or something regularly testing this on Windows.